### PR TITLE
update_params_order_of_static_method_call.php.inc fixture

### PR DESCRIPTION
**Pull request by alejomoreira**

### CODE REVIEW

File: rules-tests/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector/Fixture/update_params_order_of_static_method_call.php.inc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+final class WebUtil 
+{   
+ 	  public static function radioList($name, $items = [], $selected, $class = "", $groupingCount = 0, $groupingClass = "", $required = false)
+    {
+               
+    }
+}
+
+WebUtil::radioList("inPersonOrVirtual", $inPersonOrVirtualItems, $selected, "", 1, "col-sm-3 margin-bottom-10")
```
Comment (samsonasik) [inline rules-tests/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector/Fixture/update_params_order_of_static_method_call.php.inc]: missing `;`

```suggestion
WebUtil::radioList("inPersonOrVirtual", $inPersonOrVirtualItems, $selected, "", 1, "col-sm-3 margin-bottom-10");
```

btw, what the expected result for it? could you update PR title and description for it ?

Comment (alejomoreira) [reply rules-tests/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector/Fixture/update_params_order_of_static_method_call.php.inc]: Hey! this ir related to this discussion: https://github.com/rectorphp/rector/discussions/7397

**Comment by samsonasik** [comment]
@alejomoreira I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/2817

**Comment by TomasVotruba** [comment]
🙏 thanks

**Pull request closed without merging**